tests/compatibility/check_genesis.py

```diff
@@ -34,7 +38,7 @@ def check(old, new):
         if 'value' in old_alloc[addr]:
             continue
         # Check the code
-        if old_alloc[addr]['code'] != new_alloc[addr]['code']:
+        if check_code is True and old_alloc[addr]['code'] != new_alloc[addr]['code']:
```

Comment:
`if check_code and ..`

Reply:
That's a stupid mistake i made, i remembered it.